src/index.html

```diff
@@ -7,8 +7,46 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
```

Comment:
```suggestion
    <link 
      rel="preconnect" 
      href="https://fonts.gstatic.com"
      crossorigin="anonymous"
    >
```
Consider fixing indents according to the code style guide